README.md

```diff
@@ -1,59 +1,32 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio: setup and mobile version skeleton
 
-> One paragraph statement about the project.
+> Creating a portfolio website
```

Comment:
- [ ] Optional

- Nice description of the project. However, you can add more information on what was achieved in the project to make it more descriptive.